docs_user/modules/con_openshift-preparation-for-block-storage-adoption.adoc

```diff
@@ -34,37 +32,31 @@ spec:
 < . . . >
 ----
 +
-For more information about node selection, see xref:about-node-selector_planning[About node selector].
+For more information about node selection, see link:{defaultOCPURL}/nodes/index#nodes-scheduler-node-selectors-about_nodes-scheduler-node-selectors[About node selectors].
 +
 [NOTE]
 ====
 If your nodes do not have enough local disk space for temporary images, you can use a remote NFS location by setting the extra volumes feature ()`extraMounts`.
 ====
 Transport protocols::
-Due to the specifics of the storage transport protocols, some changes might be
-required for {OpenShiftShort}. Before making changes, review the following information:
+Some changes to the storage transport protocols might be required for {OpenShiftShort}. Before making changes, review the following information:
 +
-* Check the back-end sections in your `cinder.conf` file that are listed in the
-`enabled_backends` configuration option to determine the transport storage
-protocol that the back end uses.
+* If you use a `MachineConfig` to make changes to {OpenShiftShort} nodes, the nodes reboot.
+* Check the back-end sections that are listed in the `enabled_backends` configuration option in your `cinder.conf` file to determine the transport storage protocol that the back end uses.
```

Comment:
The two bullet points that start with “Check the back-end” and “Depending on your back-end” seem like they're both saying different ways to find the back ends. So, if the user can't find the transport storage protocol in the `enable_backends` option, they should check the `volume_driver` or `target_protocol` options?

Reply:
That's something that is poorly written (by me).  It should have said something like:

```
Check the back-end sections in your `cinder.conf` file that are listed in the `enabled_backends` configuration option to determine the enabled storage backend sections.
```

The idea is, you go and check `enabled_backends` within the `[DEFAULT]` section and you'll see one or more names, then you look for those names in the existing sections to see their configurations where you'll have the `volume_driver` configuration and optionally the `target_protocol`, and you usually can figure out the kind of storage transport protocol being used with the values assigned to those options.